<launch>
  <arg name="input_objects" default="/perception/object_recognition/objects" />
  <arg name="is_showing_debug_info" default="false"/>
  <node pkg="obstacle_avoidance_planner" type="obstacle_avoidance_planner_node"
    name="obstacle_avoidance_planner" output="screen">
    <remap from="~input/objects" to="$(arg input_objects)"/>

    <rosparam command="load" file="$(find obstacle_avoidance_planner)/config/obstacle_avoidance_planner.yaml" />

    <param name="is_showing_debug_info" value="$(arg is_showing_debug_info)"/>
  </node>
</launch>
